app/components/ui/Footer/Footer.tsx

```diff
@@ -66,32 +28,34 @@ export const Footer = () => {
                 alt="SF Department of Children Youth and their Families"
               />
             </div>
-            <address>
-              <div
-                className="site-footer__address"
-                dangerouslySetInnerHTML={{
-                  __html: htmlWithBreaks(footerData.address),
-                }}
-              />
-              <div className="site-footer__contact">
-                <a
-                  href={`tel:${callableUSPhoneNumber(footerData.phoneNumber)}`}
-                >
-                  {footerData.phoneNumber}
-                </a>
-                <br />
-                <a href={`mailto:${footerData.email}`}>{footerData.email}</a>
-              </div>
-            </address>
+            {!error && data && (
+              <address>
+                <div
+                  className="site-footer__address"
+                  dangerouslySetInnerHTML={{
+                    __html: htmlWithBreaks(data.address),
+                  }}
+                />
+                <div className="site-footer__contact">
+                  <a href={`tel:${callableUSPhoneNumber(data.phone_number)}`}>
+                    {data.phone_number}
+                  </a>
+                  <br />
+                  <a href={`mailto:${data.email_address}`}>
+                    {data.email_address}
+                  </a>
+                </div>
+              </address>
+            )}
           </div>
 
-          {footerOptions.showLinks && (
-            <div className="site-footer__links">
-              <FooterColumn column={footerData.column1} />
-              <FooterColumn column={footerData.column2} />
-              <FooterColumn column={footerData.column3} />
-            </div>
-          )}
+          <div className="site-footer__links">
+            {!error &&
+              data?.links &&
+              data.links.map((item: DynamicLink) => (
```

Comment:
Does this work the same if you combine the statements into:
```suggestion
              data?.links.map((item: DynamicLink) => (
```